### ISSUE TITLE
Lang and ARIA translatable-states-and-properties

### ISSUE THREAD

**Issue by brennanyoung** [accessibilitysupported/a11ysupport.io#225]
I just discovered that BigSur fails to honor the `lang` attribute for `aria-roledescription`. 

Repro code:
```
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8">
    <title>lang test</title>
  </head>
  <body>
  <div lang="fr">
    <button aria-roledescription="poignée HTML">Allez</button>
    <div role="button" tabindex="0" aria-roledescription="poignée ARIA">En avant</div>
  </div>
  </body>
</html>
```
Since discovering this, I'd like to see some formal tests for the `lang` attribute in relation to all the [translatable ARIA properties](https://www.w3.org/TR/wai-aria-1.2/#translatable-states-and-properties).

They are

[aria-label](https://www.w3.org/TR/wai-aria-1.2/#aria-label)
[aria-placeholder](https://www.w3.org/TR/wai-aria-1.2/#aria-placeholder)
[aria-roledescription](https://www.w3.org/TR/wai-aria-1.2/#aria-roledescription)
[aria-valuetext](https://www.w3.org/TR/wai-aria-1.2/#aria-valuetext)
